src/doc/man/cargo-fix.md

```diff
@@ -52,15 +52,37 @@ The `cargo fix` subcommand can also be used to migrate a package from one
    (without the `--edition` flag) to apply any suggestions given by the
    compiler.
 
-And hopefully that's it! Just keep in mind of the caveats mentioned above that
-`cargo fix` cannot update code for inactive features or `cfg` expressions.
-Also, in some rare cases the compiler is unable to automatically migrate all
-code to the new edition, and this may require manual changes after building
-with the new edition.
+If you encounter failing fixes, read the troubleshooting section.
 
 [edition]: https://doc.rust-lang.org/edition-guide/editions/transitioning-an-existing-project-to-a-new-edition.html
 [edition field]: ../reference/manifest.html#the-edition-field
 
+### Troubleshooting failing fixes
+
+While `cargo fix` strives to automatically apply fixes, it can still fail for
+various reasons. Here are some suggestions for troubleshooting fix failures:
+
+* Unlike other commands, `cargo fix` implies `--all-targets` by default.
+  Try fixing one Cargo target at a time instead of everything in one run.
+  For instance, narrow the scope to a single binary target:
+
+  ```sh
+  cargo fix --bin mybin
+  ```
```

Comment:
Looks like we don't have precedence for code fences like this in the man pages.

In other parts of the book, we use `console` (for interactive) instead of `sh` (for scripts)